website/docs/reference/project-collaboration-mode.md

```diff
@@ -2,52 +2,65 @@
 
 :::
 
-A project's **collaboration mode** specifies **who can submit [change requests](change-requests.md)**. There are two collaboration modes:
-- [**open**](#open-collaboration-mode)
-- [**protected**](#protected-collaboration-mode)
-- [**private**](#private-collaboration-mode)
+# Overview
 
-## **Open** collaboration mode
+Project collaboration modes allow you to manage project visibility and interaction levels, ensuring better control and collaboration within your organization.
 
-Anyone can submit change requests in an **open** project, regardless of their permissions within the project and globally. This is the default collaboration mode for projects.
+The collaboration mode of a [project](../reference/projects.md) defines who within your Unleash instance can access the project. This setting also determines who can submit [change requests](../reference/change-requests.md)–a feature that provides controlled and secure updates for feature toggles.
 
-The open collaboration mode is the default in Unleash and is how all projects worked in Unleash before the introduction of collaboration modes.
+Unleash supports three collaboration modes: **open**, **protected**, and **private**. Each mode is designed with different user roles and permissions in mind, catering to various security and collaboration requirements. Let’s explore how the three modes work.
 
-## **Protected** collaboration mode
+## Open collaboration mode
 
-Only admins and project members can submit change requests in a **protected** project. Other users can not submit change requests.
+All users of your Unleash instance can access the project and submit change requests. This is the default collaboration mode.
 
-## **Private** collaboration mode
 
-Private collaboration mode renders the project invisible to all **viewers** who are not project members. This means that **viewers** cannot see the project in the project list, nor can they access the project's features or locate the project anywhere within Unleash.
+## Protected collaboration mode
 
-**Editors**, **admins**, and users with **custom root roles** can still see private projects.
+All users of your Unleash instance can access the project but only admins and project members can submit change requests.
 
-### Change requests
 
-When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests.
+## Private collaboration mode
 
-However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to cancel the change request, but they will **not** be able to update the change request.
+All project members can access the project and submit change requests. Outside of project members, only admins, editors, and users with [custom root roles](../how-to/how-to-create-and-assign-custom-root-roles.md) can access the project and submit change requests.
```

Comment:
Just a minor comment. It will read better if there is no negative(outside of) in text. Something like
#
All project members, as well as admins, editors, and users with [custom root roles](https://github.com/Unleash/unleash/pull/how-to/how-to-create-and-assign-custom-root-roles.md), can access the project and submit change requests. Non-members do not have access.

Reply:
Thanks! What custom root role would allow you to access a private project?

Reply:
Any custom root role.

Reply:
Sorry, I meant what root permission would that role need to have? I can't see any permissions related to viewing private projects or submitting change requests.

Reply:
There's no permission to submitting change request: you need to be either be an admin or a project member.